libresonic-main/src/main/resources/liquibase/6.3/add-player-mixer.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="add-player-mixer_001" author="biconou">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="player" columnName="mixer" />
+            </not>
+        </preConditions>
+        <addColumn tableName="player">
+            <column name="mixer" type="${varchar_type}" defaultValue="">
+                <constraints nullable="true" />
```

Comment:
If this column is nullable, we shouldn't need a default value of empty string, right?